### PR TITLE
[SW-683] Remove SH tipping buttons from third party websites

### PR DESCRIPTION
**Pull request by konradodo**

### CODE REVIEW

**Review by CedrikNikita — CHANGES_REQUESTED** [review]
Commit message must not contain spelling mistakes.

**Review by CedrikNikita — COMMENTED** [review]
This is also breaking the chain name redirection script.

File: src/manifest.json [hunk 1]
```diff
@@ -20,10 +20,6 @@
     "webRequestBlocking",
     "webNavigation",
     "*://*.chain/*",
-    "*://*.google.com/*",
```
Comment (CedrikNikita) [inline src/manifest.json:23]: Removal of this line is breaking the chain name redirection script.

**Review by CedrikNikita — CHANGES_REQUESTED** [review]

**Review by peronczyk — APPROVED** [review]

**Review by CedrikNikita — CHANGES_REQUESTED** [review]
Injest.js is crucial

File: src/content-scripts/inject.js [hunk 2]
```diff
@@ -1,126 +0,0 @@
...
-          origin: window.origin,
-        },
-        origin: window.origin,
-        sendDirection: MESSAGE_DIRECTION.to_aepp,
-        receiveDirection: MESSAGE_DIRECTION.to_waellet,
-      });
-      const bridge = ContentScriptBridge({ pageConnection, extConnection });
-      bridge.run();
-    }
-  }, 10);
```
Comment (davidyuk) [inline src/content-scripts/inject.js:123]: This one should be needed for app-wallet connection

**Review by subhod-i — CHANGES_REQUESTED** [review]

File: src/manifest.json [hunk 3]
```diff
@@ -38,28 +35,8 @@
...
-      "js": ["js/twitter.js"]
-    },
-    {
-      "run_at": "document_start",
-      "matches": ["https://*.youtube.com/watch?v=*"],
-      "js": ["js/youtube.js"]
-    }
-  ],
   "web_accessible_resources": [
-    "js/inject.js",
```
Comment (CedrikNikita) [inline src/manifest.json:60]: Should be returned back.

**Review by CedrikNikita — CHANGES_REQUESTED** [review]

File: src/content-scripts/inject.js [hunk 4]
```diff
@@ -7,96 +7,6 @@ import ContentScriptBridge from '@aeternity/aepp-sdk/es/utils/aepp-wallet-commun
 window.browser = require('webextension-polyfill');
 
 const runContentScript = () => {
-  const sendToBackground = (method, params) => new Promise((resolve) => {
```
[the remark this answers is not in the document]
Comment (konradodo) [reply src/content-scripts/inject.js]: @CedrikNikita can you check it now?

**Review by subhod-i — APPROVED** [review]

File: src/store/plugins/tipUrl.js [hunk 5]
```diff
@@ -11,8 +11,7 @@ export default (store) => store.registerModule('tipUrl', {
     status: ({ verifiedUrls, blacklistedUrls }) => (tipUrl) => {
       store.dispatch('tipUrl/ensureFetched');
       if (!tipUrl) return 'default';
-      const twitterProfile = getTwitterAccountUrl(tipUrl);
-      const url = twitterProfile || tipUrl;
+      const url = tipUrl;
```
Comment (CedrikNikita) [inline src/store/plugins/tipUrl.js:14]: Why did you remove the `getTwitterAccountUrl` usage from here? It is not related to this task.

**Review by ifaouibadi — APPROVED** [review]

**Review by CedrikNikita — APPROVED** [review]

**Comment by davidyuk** [comment]
Would be good to have motivation in a written form. Otherwise, it is unclear why this decision was made from an outside view.

**Pull request merged**